src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.TaskList;
+
+public class Duke {
+
+    private TaskList taskList;
+    private Storage storage;
+    private Ui ui;
+
+    /**
+     * Constructs a duke chat bot with the given file path to store and
+     * load data from.
+     *
+     * @param filePath relative file path to data storage
```

Comment:
Javadoc statements should end in a fullstop, but honestly this is just a small thing

I just need something to write on